scenes/game/game.gd

```diff
@@ -2908,6 +2910,8 @@ func _update_buttons():
 
 	var action_buttons_visible = ui_state == UIState.UIState_PickTurnAction
 	if action_buttons_visible:
+		player_bonus_panel.visible = false
```

Comment:
Is this necessary? When is this needed that wasn't taken care of by strike cleanup/advance turn?

Reply:
I added a notice for Nine's seal effect, but she has one boost that lets her take a bonus action after sealing; I think that's the only case though, so I could just not do that if you think it would be cleaner